setup.cfg

```diff
@@ -0,0 +1,34 @@
+[metadata]
+name = ragelib
+description = A helper library for RAGE
+long_description = file: README.md
+long-description-content-type = text/markdown; charset=UTF-8
+author = Xenserver
+author_email = perf@xenserver.com
```

Comment:
I doubt this email goes anywhere other than /dev/null we can try to fix that up when we get new email addresses.

Reply:
Is  the author email field required?

Reply:
the previous email was callum's, which was going to /dev/null. In any case it seems to work fine without the author fields